index.html

```diff
@@ -25,8 +25,10 @@ <h2>How to Use</h2>
 <li>GET Landmark by City --- 
     <a href="/state/Vishakapattnam">City/Vishakahapattanam</a>
 </li>
-
-
+ <br/>
+<li>GET Landmark by City --- 
+    <a href="/state/West Bengal">City/Darjeeling</a>
```

Comment:
Please Change it as  <a href="/city/Darjeeling">City/Darjeeling</a> commit n send another PR